# HG changeset patch
# User jwe@segfault.lan
# Date 1201925835 18000
#      Fri Feb 01 23:17:15 2008 -0500
# Node ID beb698a1691628ad1fc1f52694f6c16947c6117d
# Parent  afbd31bb7b4e001c6c16d41eb908aa66bed64352
Added tag ss-2-9-13 for changeset afbd31bb7b4e

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -23,8 +23,9 @@ acb4a1e0b311c47bc957a48bf6af7f77e9c80fab
 b2e1be30c8e94902e6557c1edba2c4f37ff7bdf8 ss-2-9-7
 d542d919783961dce51ee8fc06f65b8f2cfe4156 ss-2-9-8
 60f9ced8ab534e65f27d95a712d97ba6add6cd8d ss-2-9-9
 12b676a0b18395af2fa0ccceb516447d8be6ad03 before-graphics-branch
 a848b846cb3ac279782b7fef8270a62ca4578539 ss-2-9-10
 3e2cfed6c87e2019cb1584211068b9c581331ccb before-object-branch
 10da0f6d85c260bb5f640436e42d633a57ccda9b ss-2-9-11
 db6d908e8a03a42760a45c7fd29eae074e505f7c ss-2-9-12
+afbd31bb7b4e001c6c16d41eb908aa66bed64352 ss-2-9-13
